src/Models/Game/index.js

```diff
@@ -0,0 +1,87 @@
+import {
+  CarNamesIsEmptyError,
+  TotalRoundsIsEmptyError,
+  TotalRoundsNotNumberError,
+  TotalRoundsNotIntegerError,
+  TotalRoundsNotPositiveError,
+} from "./errors";
+import { createCars } from "../Cars";
+import RandomStrategy from "../MoveStrategy/RandomStrategy";
+
+export const createGame = (carNamesInput, roundsInput) => {
+  const roundHistory = [];
+  let cars = [];
+  let totalRounds = 0;
```

Comment:
```suggestion
  const state = {
    roundHistory: [],
    cars: [],
    totalRounds: 0
  }
```
제가 리액트에 찌들어서 그런지.. 이런식으로 표현해보면 어떨까? 싶은 생각도 드네요 ㅋㅋ